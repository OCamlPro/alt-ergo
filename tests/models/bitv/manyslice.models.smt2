(set-logic ALL)
(set-option :produce-models true)
(declare-const x (_ BitVec 1))
(declare-const y (_ BitVec 1))
(declare-const xy (_ BitVec 2))
(assert (= (concat (concat x y) (concat xy xy)) (concat #b1010 (concat x y))))
(check-sat)
(get-model)
