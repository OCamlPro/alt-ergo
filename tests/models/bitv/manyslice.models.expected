
unknown
(
  (define-fun x () (_ BitVec 1) #b1)
  (define-fun y () (_ BitVec 1) #b0)
  (define-fun xy () (_ BitVec 2) #b10)
)
